scripts/init-software.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# exit script if any command fails
+set -e
+set -o pipefail
+
+# Enable submodule update for software submodules
+git config --unset submodule.software/nvdla-workload.update || :
```

Comment:
Shouldn't this be `git config --local --unset` like what the `init-submodules.sh` script does?

Reply:
https://github.com/ucb-bar/chipyard/blob/02adf86b8261ca4b5baf35b3257e15cdbabd7f87/scripts/init-submodules-no-riscv-tools-nolog.sh#L54

Reply:
I believe the `--local` flag is default.

> These values are specific to that single repository, and represent passing the --local option to git config. If you don’t specify which level you want to work with, this is the default.